content/en/docs/v3.5/op-guide/configuration.md

```diff
@@ -292,11 +290,9 @@ The security flags help to [build a secure etcd cluster][security].
 
 ### --peer-ca-file
 
-**DEPRECATED**
+**REMOVED**
 
-+ Path to the peer server TLS CA file. `--peer-ca-file ca.crt` could be replaced by `--peer-trusted-ca-file ca.crt --peer-client-cert-auth` and etcd will perform the same.
-+ default: ""
-+ env variable: ETCD_PEER_CA_FILE
+This flag has been removed. `--peer-trusted-ca-file ca.crt --peer-client-cert-auth` can be used to perform the same task as was performed by `--peer-ca-file ca.crt` previously. 
```

Comment:
```suggestion
+ This flag has been removed. `--peer-trusted-ca-file ca.crt --peer-client-cert-auth` can be used to perform the same task as was performed by `--peer-ca-file ca.crt` previously.
```
Same here, add bullet for consistency, remove extra space at the end of the line.